CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 - Deprecate the old `StringKey.discardAlertCancel` and `StringKey.discardAlertAction` string keys for overriding the discard alert buttons as they had incosistent behavior between iOS and Android ([#1001](https://github.com/Instabug/Instabug-React-Native/pull/1001)).
 - Deprecate the old `StringKey.reproStepsListItemNumberingTitle` string key for overriding the repro steps list item (screen) title as it had incosistent behavior between iOS and Android ([#1002](https://github.com/Instabug/Instabug-React-Native/pull/1002)).
 
+### Changed
+
+- Bump Instabug Android SDK to 11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)) [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0)
+
```

Comment:
_(Same changes as the iOS PR)_
Let's move the "Changed" section right after "Added" as our Changelog Guidelines document states.

Also, I have a small suggestion to align our Changelog formatting 🙏🏼
```suggestion
### Changed

- Bump Instabug Android SDK to v11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)). [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0).
```

Reply:
Thanks again for spotting this @a7medev , I've made the suggested changes, please re-check and let me know if there is any other adjustments.